### DISCUSSION TITLE
Welcome to static-site-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by ericalli** [ericalli/static-site-boilerplate#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with a question/feature/bug that's on your mind!

**Comment by devBabinski** [comment]
Hi, how pages routes are handled? There is no documentation.
Thanks
